app/conode/README.md

```diff
@@ -136,13 +141,13 @@ Once conode is compiled, you can call it with
 
 ```conode keygen address [-key keyname]```
 
-Where ```address``` is the public IP of your server. This will create two 
+Where ```address``` is the public IP of your server. This will create two
```

Comment:
Hmm, okay. I usually prefer to have in-line code formatting when referencing other code parts for consistency reasons. But the above works as well for me, whatever you usually use.